manifests/base/params-latest.env

```diff
@@ -0,0 +1,18 @@
+odh-workbench-jupyter-minimal-cpu-py311-ubi9-n=quay.io/opendatahub/workbench-images@sha256:36f48843b48b68ca8964758dad43dc0db4acb2b2d9c2f5936e1b54dc6da34019
```

Comment:
Need to update the lookup on the digest updater script to this new path and its filtering:
https://github.com/opendatahub-io/notebooks/blob/main/ci/sha-digest-updater.sh#L118

The same for the commit-latest.env

Reply:
Fixed this,please take a look.